Userland/Libraries/LibGUI/ConnectionToWindowServer.cpp

```diff
@@ -182,6 +182,24 @@ void ConnectionToWindowServer::key_down(i32 window_id, u32 code_point, u32 key,
 
     auto key_event = make<KeyEvent>(Event::KeyDown, (KeyCode)key, modifiers, code_point, scancode);
 
+    bool accepts_command_palette = true;
+    if (window->focused_widget())
+        accepts_command_palette = window->focused_widget()->accepts_command_palette();
+
+    // FIXME: This shortcut should be configurable.
+    if (accepts_command_palette && !m_in_command_palette && modifiers == (Mod_Ctrl | Mod_Shift) && key == Key_A) {
+        auto command_palette = CommandPalette::construct(*window);
+        command_palette->set_window_mode(GUI::WindowMode::CaptureInput);
+        TemporaryChange change { m_in_command_palette, true };
+        if (command_palette->exec() != GUI::Dialog::ExecResult::OK)
+            return;
+        auto* action = command_palette->selected_action();
+        VERIFY(action);
+        action->flash_menubar_menu(*window);
+        action->activate();
+        return;
+    }
+
```

Comment:
Handling command palette first looks like an artifact from the original solution, unless the intent is to prevent other actions from overriding its shortcut, in which case the commit should mention this

Reply:
True, works without relocation of the section.